packages/material-ui-lab/src/VisuallyHidden/VisuallyHidden.js

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { withStyles } from '@material-ui/core/styles';
+
+export const styles = () => ({
+  /* Styles applied to the root element. */
+  root: {
```

Comment:
Should we add these styles via the `style` prop instead of `className`, in order for making them harder to be overridden?

Reply:
The only downside I'm aware of: #19938.